storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -180,7 +180,8 @@ Options
 .. option:: --encrypt-threads=#
 
    This option specifies the number of worker threads that will be used for
-   parallel encryption. It is passed directly to the xtrabackup child process.
+   parallel encryption/decryption. It is passed directly to the xtrabackup
```

Comment:
Please remove "It is passed directly to the xtrabackup child process", it probably was copied from innobackupex reference.